packages/composite-checkout/src/components/checkout.js

```diff
@@ -261,7 +263,7 @@ function ReviewOrderStep( { isActive, isComplete, ReviewContent } ) {
 				ReviewContent ? (
 					<ReviewContent isActive={ isActive } />
 				) : (
-					<CheckoutReviewOrder isActive={ isActive } />
+					<CheckoutReviewOrder isActive={ isActive } TermsAndConditions={ TermsAndConditions } />
```

Comment:
Notice that if `ReviewContent` is rendered, `CheckoutReviewOrder` will not be, so we won't get the terms and conditions. Probably better to include it just above `CheckoutSubmitButton` as I suggested earlier.

Reply:
aah right. Good catch. I wasn't sure what that was until I created the new `WPCheckoutReviewOrder` component. With that new component, I wonder if we should just keep the terms in there and not pass it along as a prop to `Checkout`. What do you think?

Reply:
oo... That would also be a good solution! And it would keep us from having one more prop to implement.

Reply:
@sirbrillig got the fix in there. This is ready for review again.